src/bin/languageserver/mod.rs

```diff
@@ -1237,29 +1642,32 @@ impl LanguageServer for SolangServer {
     }
 
     async fn hover(&self, hverparam: HoverParams) -> Result<Option<Hover>> {
+        self.client
+            .log_message(MessageType::INFO, "hover executed!")
+            .await;
```

Comment:
Is this useful?

Reply:
I left it by mistake, I have removed it now. Thanks!